src/components/Events/PublishedScheduleTable.jsx

```diff
@@ -0,0 +1,120 @@
+import { NPOBackend } from '../../utils/auth_utils.js';
+import Events from './Events.jsx';
+import EventInfo from './EventInfo.jsx';
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+import { Table, Thead, Tbody, Tr, Th, Td, TableContainer, Box, IconButton, useDisclosure } from '@chakra-ui/react';
+import AddDayModal from '../../pages/PublishedSchedule/AddDayModal.jsx'
+import { AddIcon } from '@chakra-ui/icons';
+import { useAuthContext } from '../../common/AuthContext.jsx';
+import AUTH_ROLES from '../../utils/auth_config.js';
+const { ADMIN_ROLE } = AUTH_ROLES.AUTH_ROLES;
+
+
+const PublishedScheduleTable = ({ season }) => {
+  const {currentUser} = useAuthContext();
+
+  const [eventsInDay, setEventsInDay] = useState([]);
+  const seasonType = season.split(' ')[0];
+  const seasonYear = season.split(' ')[1];
+  const [dataShouldRevalidate, setShouldDataRevalidate] = useState(false);
+  const { isOpen: isOpenDay, onOpen: onOpenDay, onClose: onCloseDay } = useDisclosure();
+
+  const renderTable = async () => {
+    const { data } = await NPOBackend.get(
+      `/published-schedule/season?season=${seasonType}&year=${seasonYear}`,
+    );
+    setEventsInDay(data);
+  };
+
+  useEffect(() => {
+    renderTable();
+  }, [seasonType, seasonYear]);
+
+  useEffect(() => {
+    if (dataShouldRevalidate) {
+      renderTable();
+      setShouldDataRevalidate(false);
+    }
+  }, [dataShouldRevalidate])
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'renderTable'. Either include it or remove the dependency array.